R/qualification-ddi.R

```diff
@@ -48,12 +49,18 @@ getQualificationDDIPlotData <- function(configurationPlan) {
 
         for (pkParameter in pkParameters) {
           ratioList[[pkParameter]] <- list()
-
           validateIsIncluded(ddiPKRatioColumnName[[pkParameter]], names(observedDataFrame))
-          ratioList[[pkParameter]]$observedRatio <- observedDataFrame[[ddiPKRatioColumnName[[pkParameter]]]][observedDataFrame$ID %in% observedDataRecordId]
+          ratioList[[pkParameter]]$id <- observedDataFrame[["ID"]][observedDataFrame$ID %in% observedDataRecordId]
+          ratioList[[pkParameter]]$studyId <- observedDataFrame[["Study ID"]][observedDataFrame$ID %in% observedDataRecordId]
           ratioList[[pkParameter]]$mechanism <- observedDataFrame[["Mechanism"]][observedDataFrame$ID %in% observedDataRecordId]
           ratioList[[pkParameter]]$perpetrator <- observedDataFrame[["Perpetrator"]][observedDataFrame$ID %in% observedDataRecordId]
+          ratioList[[pkParameter]]$routePerpetrator <- observedDataFrame[["Route Perpetrator"]][observedDataFrame$ID %in% observedDataRecordId]
+          ratioList[[pkParameter]]$routeVictim <- observedDataFrame[["Route Victim"]][observedDataFrame$ID %in% observedDataRecordId]
           ratioList[[pkParameter]]$victim <- observedDataFrame[["Victim"]][observedDataFrame$ID %in% observedDataRecordId]
+          ratioList[[pkParameter]]$dose <- observedDataFrame[["Dose"]][observedDataFrame$ID %in% observedDataRecordId]
+          ratioList[[pkParameter]]$doseUnit <- observedDataFrame[["Dose Unit"]][observedDataFrame$ID %in% observedDataRecordId]
+          ratioList[[pkParameter]]$description <- observedDataFrame[["Description"]][observedDataFrame$ID %in% observedDataRecordId]
+          ratioList[[pkParameter]]$observedRatio <- observedDataFrame[[ddiPKRatioColumnName[[pkParameter]]]][observedDataFrame$ID %in% observedDataRecordId]

```

Comment:
No idea what `observedDataFrame$ID %in% observedDataRecordId` means, but can't it be assigned to a variable first? 
e.g.
```
XYZ <- observedDataFrame$ID %in% observedDataRecordId
ratioList[[pkParameter]]$id <- observedDataFrame[["ID"]][XYZ]
ratioList[[pkParameter]]$studyId <- observedDataFrame[["Study ID"]][XYZ]
...
```